core/src/main/java/apoc/export/csv/ExportCSV.java

```diff
@@ -84,10 +84,11 @@ public Stream<ExportProgressInfo> all(
                     {
                             stream = false :: BOOLEAN,
                             batchSize = 20000 :: INTEGER,
-                            bulkImport = true :: BOOLEAN,
+                            bulkImport = false :: BOOLEAN,
```

Comment:
Note to reviewer, I didn't change the default, it was this signature that was wrong :)